docs/node-operator-questionnaire.md

```diff
@@ -0,0 +1,53 @@
+# Node Operators Questionnaire
```

Comment:
Maybe put in some "Hardware Requirements" that are relatively equal to the CPU / mem reservations of our nilDB containers running on ECS?

Reply:
I did think about this ... is a specific cpu or memory count really what we're after?  I think at the moment our nildb nodes run on 1vcpu + 1 gb ram which we should actually bump when mainnet so I don't think those are good values to suggest here.

What I attempted to capture in `Compute Resources` was for them to explain how they scale to meet demand which imo is a better indicator.

Reply:
I reviewed the Compute Resource section. Lgtm.